src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -31,6 +31,9 @@ interface VcsRootLocator {
 }
 
 interface BuildLocator {
+    fun withId(buildId: BuildId): BuildLocator
```

Comment:
We already have methods which find build by id and by build number directly in TeamCityInstance. I think it makes no sense to have these method in BuildLocator because there may be at most one build with the specified id or build number, so it wouldn't make sense to apply these methods together with other filtering methods.